pkg/awx/project_resource.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This file contains the implementation of the resource that manages a specific project.
+
+package awx
+
+import (
+	"github.com/openshift/autoheal/pkg/awx/internal/data"
+)
+
+type ProjectResource struct {
+	Resource
+}
+
+func NewProjectResource(connection *Connection, path string) *ProjectResource {
+	resource := new(ProjectResource)
+	resource.connection = connection
+	resource.path = path
+	return resource
+}
+
+func (r *ProjectResource) Get() *ProjectGetRequest {
+	request := new(ProjectGetRequest)
+	request.resource = &r.Resource
+	return request
+}
+
+type ProjectGetRequest struct {
+	Request
+}
+
+func (r *ProjectGetRequest) Send() (response *ProjectGetResponse, err error) {
+	data := new(data.ProjectGetResponse)
```

Comment:
`data` is a bit confusing with the name of the package. I suggest `output` - this is what we use for `JobTemplatesResource`, `JobTemplateLaunchResource`, `JobResource` and `JobsResource`.

If you agree, you might as well change `JobTemplatesResource`, which also uses `data`

Reply:
Yes, I agree, will do.